src/utils.ts

```diff
@@ -526,13 +529,14 @@ export interface ClientMetadata {
     version: string;
   };
   platform: string;
```

Comment:
This was wrong, according to the spec: https://github.com/mongodb/specifications/blob/master/source/mongodb-handshake/handshake.rst#hello-command

this modifies a public interface but it's also technically a bug fix, so I think it's okay to remove